persistence/src/main/java/com/stezhka/dao/jdbc/mapper/MovieRandomRowMapper.java

```diff
@@ -0,0 +1,53 @@
+package com.stezhka.dao.jdbc.mapper;
+
+import com.stezhka.entity.Country;
+import com.stezhka.entity.Genre;
+import com.stezhka.entity.Movie;
+import org.springframework.jdbc.core.RowMapper;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MovieRandomRowMapper implements RowMapper<Movie> {
+
+    @Override
+    public Movie mapRow(ResultSet resultSet, int i) throws SQLException, NullPointerException {
+        Movie movie = new Movie();
+
+        movie.setId(resultSet.getInt("id"));
+        movie.setNameRussian(resultSet.getString("name_russian"));
+        movie.setNameNative(resultSet.getString("name_native"));
+        movie.setYearOfRelease(resultSet.getInt("year_of_release"));
+        movie.setDescription(resultSet.getString("description"));
+        movie.setRating(resultSet.getDouble("rating"));
+        movie.setPrice(resultSet.getDouble("price"));
+        movie.setPoster(resultSet.getString("picture_path"));
+        movie.setVotes(resultSet.getInt("votes"));
+        movie.setCountries(getMovieCountries(resultSet));
+        movie.setGenres(getMovieGenres(resultSet));
+
+        return movie;
+    }
+
+    private List<Country> getMovieCountries(ResultSet resultSet) throws SQLException, NullPointerException {
```

Comment:
NullPointerException I belive redundant or should be handled

Reply:
Fixed